ting-bill-split.go

```diff
@@ -62,7 +128,9 @@ func main() {
 		log.Fatal(err)
 	}
 
-	readAndPrint(minFile)
 	readAndPrint(msgFile)
 	readAndPrint(megFile)
+
+	minMap := parseMinutes(bufio.NewReader(minFile))
```

Comment:
`minFile` is an `os.File` which already implements `io.Reader` so you don't need to pass it to `bufio`